test/sql/append.sql

```diff
@@ -0,0 +1,110 @@
+\ir include/create_single_db.sql
+
+-- create a now() function for repeatable testing that always returns
```

Comment:
FWIW, also simplified the way diff tests are performed, hence the large number of changed files as it required a change in the test database setup.